dashboard/src/containers/Helpers.js

```diff
@@ -547,3 +547,11 @@ export function dgraphAddress() {
   // For development, we just connect to the Dgraph server at http://localhost:8080.
   return "http://localhost:8080";
 }
+
+export function dgraphQuery(debug = true) {
```

Comment:
IDEA: in the future we could make an even more general factory function for routes. But I think it's fine for now.